src/Types/formFields.ts

```diff
@@ -0,0 +1,5 @@
+export interface FormFields {
+  [key: string]: string;
```

Comment:
I think you have only a title and selectedUserId fields, not a random strings and the same with `ErrorsFields`